src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/listeners/ListenerHelpers.kt

```diff
@@ -23,5 +23,8 @@ data class SQSMessage(val Type: String, val Message: String, val MessageId: Stri
 data class MessageAttributes(val eventType: EventType)
 data class EventType(val Value: String, val Type: String)
 
-fun asCompletableFuture(process: suspend () -> Unit): CompletableFuture<Void> =
-  CoroutineScope(Context.current().asContextElement()).future { process() }.thenAccept { }
```

Comment:
I think the `thenAccept{}` here was used to ensure we got a `Void` back, but can do that without chaining the future